front/src/components/AddMockupSurvey.js

```diff
@@ -0,0 +1,455 @@
+import React, {Component} from "react";
+import {Button, ButtonDropdown, Card, CardBody, CardFooter, CardHeader, Col, DropdownItem, DropdownMenu,
+    DropdownToggle, Form, FormGroup, Input, Label, Row} from 'reactstrap';
+import Fullscreen from "react-full-screen";
+import "../App.css";
+import AuthenticationService from "../service/AuthenticationService";
+import {Redirect} from "react-router-dom";
+import axios from 'axios'
+import * as Survey from "survey-react";
+
+class AddMockupSurvey extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            serverUrl: "http://localhost:8080",
+            mockupId: this.props.match.params.id,
+            mockup: "",
+            surveyName: "",
+            questions: [],
+            iframeWidth: '125%',
+            iframeHeight: window.innerHeight,
+            scale: 'scale(0.80)',
+            isFull: false,
```

Comment:
Probably would have changed name isFull to isFullscreen. isFull might mean different things. isFullscreen is more precise